index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>tribute page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <main id="main">
+        <h1 id="title">Dr. Noman Bourlaug</h1>
+        <p id="discp">The man who saved a billion lives</p>
+        <figure id="image-div">
+            <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists">
+         <figcaption id="image-caption">
+            Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.
+         </figcaption>
+        </figure>
+        <div id="tribute-info">
+            <h2 id="subtitle">
+                Here's a time line of Dr. Borlaug's life:
+            </h2>
+            <ul> <li>
+               <strong>1914</strong>  - Born in Cresco, Iowa
+            </li>
+            <li>
+                <strong>1933</strong>  - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration"
+             </li>
+             
```

Comment:
Please phrase the comments politely, as suggestions instead of orders.
For example:

This is a polite suggestion: "I think it would be better to use semantic tags"
This reads as an order: "Use semantic tags"

Kindly always include an explanation, a "why", for each requested change. For example:

* This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
* This request lacks a "why": *Why don't you use an article tag here instead of a div?*

Reply:
As requested by the previous reviewer, Kindly add the reason why you ask the student to avoid extra spaces. Giving the reason prevents any sort of confusion recycle